docs/developers/cosmwasm-contract-deployment.md

```diff
@@ -54,7 +56,13 @@ Let's now deploy our smart contract!
 Run the following:
 
 ```sh
-TX_HASH=$(wasmd tx wasm store artifacts/cw_nameservice.wasm --from $KEY_NAME --keyring-backend test $TXFLAG --output json -y | jq -r '.txhash') 
+TX_HASH=$(wasmd tx wasm store artifacts/cw_nameservice.wasm --from $KEY_NAME --keyring-backend test $TXFLAG $NODEIP --output json -y | jq -r '.txhash')
+```
+
+Your output will be the gas estimate:
+
+```sh
+gas estimate: 1344759
```

Comment:
oh weird, as is, the command `TX_HASH=$(wasmd tx wasm store artifacts/cw_nameservice.wasm --from $KEY_NAME --keyring-backend test $TXFLAG $NODEIP --output json -y | jq -r '.txhash')` returns a gas estimate for me. if i prompt to return tx hash, it works. adding `&& echo $TX_HASH` to the command